hare.spec

```diff
@@ -45,7 +45,7 @@ BuildRequires: python36-devel
 BuildRequires: python36-pip
 BuildRequires: python36-setuptools
 
-Requires: consul = 1.7.8
+Requires: consul = 1.9.1
```

Comment:
Also change needed in :
File README.md
Line 59 : sudo yum -y install consul-1.7.8

Reply:
Ah, excellent catch! thanks.

Reply:
Done.

Reply:
@mssawant please have a look at #1475. Perhaps that approach is even better?